client/src/main/scala/com/github/dapperware/slack/api/SlackPins.scala

```diff
@@ -0,0 +1,13 @@
+package com.github.dapperware.slack.api
+
+import com.github.dapperware.slack.{ SlackEnv, SlackError }
+import zio.ZIO
+
+
+trait SlackPins {
+  def pin(channelId: String, timeStamp: Option[String] = None): ZIO[SlackEnv, SlackError, Boolean] = {
+    val channelHeader = "channel" -> channelId
+    val headers = timeStamp.fold(Seq(channelHeader))(ts => Seq(channelHeader, "timestamp" -> ts))
+    sendM(requestQueryParams("pins.add", headers)) >>= isOk
```

Comment:
I'd prefer to use `json` here which should be supported according to the docs: https://api.slack.com/methods/pins.add

Reply:
you mean the return type as `ZIO[SlackEnv, SlackError, Json]` instead of `ZIO[SlackEnv, SlackError, Boolean]`?